kickstarts/partials/main/bootloader.ks.erb

```diff
@@ -5,5 +5,5 @@ bootloader --location=mbr --driveorder=vda --append="crashkernel=auto rhgb quiet
 <% elsif @target == "gce" %>
 bootloader --location=mbr --driveorder=vda --append="crashkernel=auto rhgb quiet net.ifnames=0 biosdevname=0 console=ttyS0,38400n8"
 <% else %>
-bootloader --location=mbr --driveorder=vda --append="crashkernel=auto rhgb quiet net.ifnames=0 biosdevname=0"
+bootloader --location=mbr --driveorder=vda --append="crashkernel=auto rhgb quiet net.ifnames=0 biosdevname=0 console=ttyS0,115200 console=tty0"
```

Comment:
Should this be conditional to kubevirt?  Or asking another way how does this affect non-kubevirt usage?

Reply:
`tty0` is the default and since it's listed last, it will continue to be the default.  Adding `ttyS0` enables the serial console as a secondary console.